particles/Processing/TopLabels.particle

```diff
@@ -0,0 +1,10 @@
+import './schemas/MlResource.schema'
+import './schemas/MlModel.schema'
+import './schemas/Inference.schema'
+
+particle TopLabels in './js/TopLabels.js'
+  in TfYHat yHat
```

Comment:
Why is there a type for `TfYHat`? 
These should be `TfTensor`s.

Reply:
Ok, I simply didn't know.

Reply:
If you want to have different types to take advantage of matching, I would prefer another means, like tagging. 
This just rubs me the wrong way in terms of ML conventions. It's kinda like saying that there is a type, `Array`, and then another type `OutputArray`, where you're supposed to use the latter only for the return of functions. But really, both are arrays.

Besides just rubbing me the wrong way, this limits what operations we are able to call. I should be able to call `Normalize` or `Reshape` on the output of `Infer`, for instance.

Reply:
The distinct types should only be for distinct types. I just didn’t realize yHat was a Tensor.

If there are other mistakes we’ll fix them.